src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -247,8 +247,12 @@ type collectionAssertions map[string]collectionAssertion
 // 	}
 // }
 
-func rootAnd(items ...models.DriveItemable) []models.DriveItemable {
-	return append([]models.DriveItemable{driveItem(rootID, rootName, parent(0), "", isFolder)}, items...)
+func pageItems(items ...models.DriveItemable) []models.DriveItemable {
```

Comment:
doesn't need to be done in this PR, but at some point we should consolidate the helper functions into a single file. Right now we're using helpers defined in both this file and `collections_test.go` (maybe others too..?)

Reply:
Test cleanup begins happening here and continues, interpolated with the other changes, across a series of PRs.  This suggestion is implemented in the next one, iirc.